spec/_layouts/default.html

```diff
@@ -32,76 +31,95 @@
+           <img height="50" alt="visit the hl7 website" width="42" src="{{base}}dist/hl7-logo.png"/>
+				</a>
+			</div>
+      <!--
+			<div id="hl7-nav"><a id="hl7-logo" no-external="true" href="http://hl7.org/fhir/search.cfm"><img alt="Search FHIR" src="dist/search.png"/></a></div>
+        -->
 		</div>
 		<div class="container">  <!-- container -->
-	</div></div>  <!-- /segment-header -->	
+	</div></div>  <!-- /segment-header -->
 
 	<div id="segment-navbar" class="segment">  <!-- segment-navbar -->
 		<div id="stripe"> </div>
 		<div class="container">  <!-- container -->
    <!-- HEADER CONTENT -->
- 	      		
- 	      		      		
+
+
 			<nav class="navbar navbar-inverse">
 				<div class="container">
 					<button data-target=".navbar-inverse-collapse" class="navbar-toggle" data-toggle="collapse" type="button">
 						<span class="icon-bar"> </span>
 						<span class="icon-bar"> </span>
 						<span class="icon-bar"> </span>
 					</button>
-					<a class="navbar-brand hidden" href="index.html">FHIRPath</a>
+					<a class="navbar-brand hidden" href="#">Navbar</a>
 					<div class="nav-collapse collapse navbar-inverse-collapse">
 						<ul class="nav navbar-nav">
-              <li><a href="index.html">Documentation</a></li>
-              <li><a href="grammar.html">Grammar</a></li>
-              <li><a href="tests.html">Tests</a></li>
-              <li><a href="history.html">Version History</a><!--current--></li>
+              <li><a href="{{base}}index.html">Home</a></li>  <li class="dropdown">
+                  <a href="#" data-toggle="dropdown" class="dropdown-toggle">Conformance<b class="caret">
+                  </b>
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a href="{{base}}conformance/index.html">Declaring Conformance</a></li>
+                  <li><a href="{{base}}conformance/core-set/index.html">Conformance: Core Set</a></li>
+                  <li><a href="{{base}}conformance/index.html">Conformance: Capabilities Sets</a></li>
+                  <li><a href="{{base}}capability-statement/index.html">Syntax for declaring Conformance</a></li>
+                </ul>
+              <li><a href="{{base}}scopes-and-launch-context/index.html">Scopes and Launch Context</a><!--current--></li>
 
 						</ul>
 					</div>  <!-- /.nav-collapse -->
 
           </div>  <!-- /.container -->
 			</nav>  <!-- /.navbar -->
-      		
-				
-  <!-- /HEADER CONTENT -->				
+
+
+  <!-- /HEADER CONTENT -->
 		</div>  <!-- /container -->
 	</div>  <!-- /segment-navbar -->
-	
+
 	<div id="segment-content" class="segment">  <!-- segment-content -->
 	<div class="container">  <!-- container -->
             <div class="row">
             	<div class="inner-wrapper">
   <!-- CONTENT CONTENT -->
 
 
-<div class="col-9">
-    
+<div class="col-12">
+
+<!-- Maturity level Box -->
+
+<!--
```

Comment:
If this commented code is no longer used, let's delete?

Reply:
OK